app/controllers/course/user_invitations_controller.rb

```diff
@@ -18,10 +18,15 @@ def create # :nodoc:
   private
 
   def course_user_invitation_params # :nodoc:
-    invitations_attributes = { course_user: [:name], user_email: [:email] }
-    @invite_params ||= params.require(:course).
-                       permit(:invitations_file, :registration_key,
-                              invitations_attributes: invitations_attributes)
+    @invite_params ||= begin
```

Comment:
oops, I should have called this `@course_user_invitation_params`. Can help me change please?